docs/Home/index.js

```diff
@@ -23,6 +37,25 @@ const Foo = () => (
 );
       `}
     </Code>
+    <h2>Contribute</h2>
+    <p>
+      Check out our{' '}
+      <Link href="https://github.com/dequelabs/cauldron/blob/develop/CONTRIBUTING.md">
+        contribution guide!
+      </Link>
```

Comment:
We could use an icon with `type="external-link"` here.

```suggestion
      <Link href="https://github.com/dequelabs/cauldron/blob/develop/CONTRIBUTING.md">
        <Icon type="external-link" />
        contribution guide!
      </Link>
```

https://cauldron.dequelabs.com/components/Icon